src/Control/Lens/Plated.hs

```diff
@@ -102,9 +102,6 @@ import Control.Lens.Traversal
 import Control.Monad.Free as Monad
 import Control.Monad.Free.Church as Church
 import Control.Monad.Trans.Free as Trans
-#if !(MIN_VERSION_free(4,6,0))
```

Comment:
Shouldn't this have been made unconditional rather than removed?

Reply:
The lower version bounds on `free` are currently:

https://github.com/ekmett/lens/blob/d47860c8498d4db20c8a17756a5d4c52be256683/lens.cabal#L187

As a result, this code can never be reached, unless I'm overlooking something.

Reply:
No, you're right.